dhctl/pkg/kubernetes/actions/deckhouse/log.go

```diff
@@ -82,15 +85,43 @@ func isErrorLine(line *logLine) bool {
 		return true
 	}
 
+	// Consider stderr messages are errors too.
 	if line.Output == "stderr" {
-		// skip tiller output
-		if line.Component == "tiller" {
-			return false
+		// Skip possible messages from tiller.
+		if line.Component != "tiller" {
+			return true
 		}
```

Comment:
Deckhouse use Helm3 since at least 1.26 (Dec 2020) (See https://github.com/deckhouse/deckhouse/releases/tag/v1.26.0). May be it is time to remove this condition?

Reply:
why not